### ISSUE TITLE
Part 4: reduce server response size

### ISSUE THREAD

**Issue by benoitdm-oslandia** [opengeospatial/ogcapi-features#738]
For features with big geometry, what kind of mechanism do you suggest to apply to avoid the server sends the whole geometry when the user ask for `{landingPageUri}/collections/{collectionId}/items`? This is also the case when the user GET a single feature or PATCH a feature with only one property (says `color`) and receive in response the whole geometry WKT.

Is there some kind of Level Of Detail mechanism or a parameter on the request to specify the wanted properties in response?

**Comment by benoitdm-oslandia** [comment]
@cportele Great proposals! 

May I suggest to avoid negate word in var name as [it tends to bring confusion](https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables): you could replace `skipGeometry` by `addGeometry` or `enableGeometry`.

I thinks we will implement these extensions in our next developments.

**Comment by cportele** [comment]
@benoitdm-oslandia 

> avoid negate word

Both options are not really optimal (skip=true fails "avoid negatives in names for standalone variables" and enable=false fails "always choose names that enable setting `false` by default"). Anyhow, from initial discussions it seems that most would prefer to treat the geometry property like any other property and include it in properties (or not).

Thank you for the feedback!

**Comment by benoitdm-oslandia** [comment]
@cportele you're right! It looks like a better solution to treat the geometry property like any other property!

Thanks!

**Comment by benoitdm-oslandia** [comment]
Hi @cportele,

I am rolling back my thoughts for the property selection, do you think it will be possible to have an `excluded-properties` parameter? 

It can be useful when only a small sub set of properties needs to be avoided instead to list all displayable properties (as it can be a long list depending on the feature). 

This property will be or-exclusive with the `properties` parameter.

**Comment by cportele** [comment]
Meeting 2024-06-03: One part of the request will be addressed by Part 6 ("parameter on the request to specify the wanted properties in response") the other by Part 7 ("Level Of Detail mechanism").

What is not yet included in the draft of Part 6 is the option to omit selected properties. For cases where features have a lot of properties and you just want to omit a few, it is easier to specify just the few to be omited. We will add a `exclude-properties` parameter. The `properties` and `exclude-properties` parameters will be mutually exclusive.
